doc/irmin/index.mld

```diff
@@ -1,32 +1,32 @@
 {0 Irmin}
 
-[Irmin] is a distributed database built on the same principles as {{:https://git-scm.com} Git}. It can be used as an OCaml library that provides persistent stores with built-in snapshot, branching and reverting mechanisms.
+[Irmin] is a distributed database built on the same principles as {{:https://git-scm.com} Git}. Specifically designed for [the MiragoOS ecosystem](https://mirage.io/), it can be used as an OCaml library that provides persistent stores with built-in snapshot, branching, merging, history trackig, and reverting mechanisms, similar to a version control system like Git, but for data rather than code.
```

Comment:
```suggestion
[Irmin] is a distributed database built on the same principles as {{:https://git-scm.com} Git}. It can be used as an OCaml library that provides persistent stores with built-in snapshot, branching, merging, history tracking, and reverting mechanisms, similar to a version control system like Git, but for data rather than code.
```

I'm not sure about "Specifically designed for MirageOS", since we wouldn't want to send the message that Irmin can only be used with Mirage? (but the typo `Mirago` is such a cute name :blush:)

Reply:
Thanks for catching those typos! Good eye!